src/11.numberformat.js

```diff
@@ -433,16 +435,16 @@ function GetFormatNumber() {
         return internal['[[boundFormat]]'];
     }
 
-function formatToParts(value = undefined) {
+function formatToParts() {
   let internal = this !== null && typeof this === 'object' && getInternalProperties(this);
   if (!internal || !internal['[[initializedNumberFormat]]'])
       throw new TypeError('`this` value for formatToParts() is not an initialized Intl.NumberFormat object.');
 
-  let x = Number(value);
+  let x = Number(arguments[0]);
```

Comment:
@JakeChampion any reason for this change?

Reply:
was is to avoid using the new syntax?

Reply:
This was because the function length of `Intl.NumberFormat.prototype.formatToParts` is meant to be 0 according to the test262 tests.

https://github.com/andyearnshaw/Intl.js/pull/288/files#diff-c4bc087762cdd9abd6e0456570110b11R329